src/main/java/org/commonjava/util/sidecar/metrics/MetricFieldsConstants.java

```diff
@@ -0,0 +1,22 @@
+package org.commonjava.util.sidecar.metrics;
+
+public class MetricFieldsConstants {
+
+    public static final String HTTP_METHOD = "http-method";
+
+    public static final String STATUS_CODE = "status-code";
+
+    public static final String PATH_INFO = "path-info";
+
+    public static final String PROXY_TRACE_ID = "proxy-trace-id";
```

Comment:
maybe we just name it "trace-id". sidecar initiates this id, add to headers and pass to indy server. 
we also need to add another field "service" with value "sidecar". Eventually all services send data to a same dataset (so we can query by the 'trace-id'). That is why we need a field to identify what entry is from what service.

Reply:
This is just used for span field naming: proxy-trace-id.
For header transfer:
Indy client also has a trace-id to transfer into the server, and also incorporate it into the server root span eventually.
So just naming "trace-id" in header will make confused what's the trace id used for and where does it come from.
I used "Proxy-Trace-Id" header for Proxy, "Indy-Client-Trace-Id" header for Client, both together with the paired "Proxy-Span-Id" or "Indy-Client-Span-Id" will make us easy to analyze on the server side metrics.
That's my thought, I'm not sure if you indicates the naming of header or just metrics, let's discuss.

Reply:
I suppose we will use sth like: select from datasetA where trace-id=123, the expect result:
trace-id | service | fields...
-----------------------------
123,sidecar, ...
123,content-service, ...

If we use different field name for the id, we have to do: select * where trace-id=xxx or proxy-trace-id=xxx or ...
So we'd better use a single field for the id, and other fields like 'service', 'function' to differentiate where it come from.

Reply:
@ruhan1 Agree, it looks this is only for metric query.